commands.js

```diff
@@ -740,7 +740,7 @@ var commands = exports.commands = {
 			return this.sendReply("User " + this.targetUsername + " not found.");
 		}
 		if (Rooms.rooms[targetRoom.id].users[targetUser.userid]) {
-			return this.sendReply("User " + targetUser.name + " is already in the room " + targetRoom.title + "!");
+			return this.sendReply("User " + targetUser.name + " is already in the room " + (targetRoom.title || "global") + "!");
```

Comment:
I guess it would be safer than just assuming that the room would always be global.